src/main/java/org/spongepowered/api/item/inventory/MultiBlockCarrier.java

```diff
@@ -0,0 +1,27 @@
+package org.spongepowered.api.item.inventory;
+
+import org.spongepowered.api.world.Location;
+import org.spongepowered.api.world.World;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Represents Multiple Blocks that are the Carrier for an Inventory.
+ * e.g. A DoubleChest Inventory carried by two blocks
+ */
+public interface MultiBlockCarrier extends Carrier {
```

Comment:
Is it possible to inherit `BlockCarrier`? rename it to `SingleBlockCarrier` and add a parent `BlockCarrier`?